src/storage/service.ts

```diff
@@ -494,10 +507,11 @@ export class StorageService {
 
     // Filter for this provider's proof sets
     const providerProofSets = proofSets.filter(
-      ps => ps.payee.toLowerCase() === provider.owner.toLowerCase() &&
-            ps.isLive &&
-            ps.isManaged &&
-            ps.withCDN === withCDN
+      ps =>
+        ps.payee.toLowerCase() === provider.owner.toLowerCase() &&
```

Comment:
is your editor doing this? best to keep changes like this to a minimum where possible to make reviewing a bit easier